controller/applicationsController.js

```diff
@@ -0,0 +1,67 @@
+const Admin = require("../models/admins");
+const Organization = require("../models/organizations");
+const Application = require("../models/applications");
+const { getAppToken } = require("../utils/auth/tokenGenerator");
+const { ObjectId } = require("mongoose").Types;
+
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+
+const createApplication = async (req, res, next) => {
+  try {
+    //validation should be done via middleware
+    //ownerId in body also needs to be validated
+
+    const { name, organizationId, createdBy } = req.body;
+
+    if (!ObjectId.isValid(organizationId) || !ObjectId.isValid(createdBy)) {
+      next(new CustomError(404, "invalid ID"));
+      return;
+    }
+
+    if (!name || !organizationId || !createdBy) {
```

Comment:
This is good but not entirely necessary as the validation middleware would have already caught this